curriculum/js1/10.js

```diff
@@ -7,7 +7,9 @@
  * @param {function} c
  */
 
-const solution = (a, b, c) => {
+const solution = (a,b,c) => {
+  setTimeout(c,a*1000);
+  setTimeout(c,b*1000);
```

Comment:
This second call of function `c` must execute `b` seconds after the first execution of `c`.